pumps/mongo.go

```diff
@@ -164,9 +165,14 @@ func mongoType(session *mgo.Session) MongoType {
 
 	if result.Code == 303 {
 		return AWSDocumentDB
-	} else {
-		return StandardMongo
 	}
+
+	if result.Code == 115 {
+		return CosmosDB
+	}
```

Comment:
@mativm02 could you explain why 115 is CosmoDB?

Reply:
> @mativm02 could you explain why 115 is CosmoDB?

I saw that we were using the result code to check whether the connection is over DocumentDB or not. So I check what was the returned code when connecting to CosmosDB and it is 115. 
I tried to find a way to get some explicit information about the session (if it's connected to a simple Mongo, DocumentDB, or CosmosDB) but didn't find anything.

The method session.LiveServers() returns a part of the connection URL, in this case: `4-0-qa.mongo.cosmos.azure.com:10255`, I think we could check what's the DB server with it,  but I am not sure if this will work for all connections.

Do you know about another better option?